modules/context/access_log.go

```diff
@@ -20,20 +22,51 @@ type routerLoggerOptions struct {
 	Start          *time.Time
 	ResponseWriter http.ResponseWriter
 	Ctx            map[string]interface{}
+	RequestID      *string
 }
 
 var signedUserNameStringPointerKey interface{} = "signedUserNameStringPointerKey"
 
+const keyOfRequestIDInTemplate = ".RequestID"
+
+// According to:
+// In OpenTracing and OpenTelemetry, the maximum length of trace id is 256 bits (32 bytes).
+// MD5 output is 16 or 32 bytes.
+// UUID output is 36 bytes (including four ‘-’)
+// SHA1 output is 40 bytes
+// So, we accept a Request ID with a maximum character length of 40
```

Comment:
Some comments are not accurate.

* "256bit", it causes misunderstanding, if you are using 32bytes (byte=0~255) then it's true. But if you are using ASCII chars (like hex/base64, etc), then the information entropy is less than 256bit. According to the document:
    *  `TraceId`: A valid trace identifier is a 16-byte array with at least one non-zero byte. So its hex string is 32bytes (true for OpenTracing/OpenTelemetry)
* `MD5 output is 16 or 32 bytes`: md5-bytes is 16, md5-hex is 32
* SHA1: similar, SHA1-bytes is 20, SHA1-hex is 40.
* However, SHA1 and MD5 are all deprecated, it's not worth to mention them nowadays IMO
* UUID is 128-bit, 32 hex chars, 36 ASCII chars with 4 dashes